.github/workflows/push.yml

```diff
@@ -7,24 +7,15 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [macos-latest, windows-latest, ubuntu-latest]
```

Comment:
Can we remove this ubuntu-latest, I see there is a expected `build (ubuntu-latest)` waiting in the ci.

Reply:
Yup, just need to merge this

Reply:
Okey, Thanks.